linuxclientsetup/config-files/firefox/prefs.cfg

```diff
@@ -0,0 +1,18 @@
+//
+lockPref("xpinstall.enabled", false);
+lockPref("browser.cache.disk.capacity", 5120);
+pref("extensions.zoteroOpenOfficeIntegration.installed", true);
+pref("extensions.zoteroOpenOfficeIntegration.skipInstallation", true);
+pref("extensions.zotero.integration.useClassicAddCitationDialog", true);
+pref("app.update.enabled", false);
+pref("browser.search.selectedEngine", "Google");
+pref("browser.search.suggest.enabled", false);
+pref("browser.tabs.loadInBackground", true);
+pref("browser.tabs.loadBookmarksInBackground", true);
```

Comment:
Perhaps we should remove these two `browser.tabs` settings from the default? They affect UX, and we should be shipping defaults with the ability to change it.